Sources/Bridges/OTelSwiftLog/LogHandler.swift

```diff
@@ -0,0 +1,192 @@
+import Foundation
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import Logging
+
+// let the bridgename be the url of the package?
+let bridgename: String = "OTelSwiftLog"
+let version: String = "1.0.0"
+
+public struct InstrumentationScope{
+    public var name: String
+    public var version: String?
+    public var eventDomain: String?
+    public var schemaUrl: String?
+    public var includeTraceContext: Bool?
+    public var attributes: [String: AttributeValue]?
+}
+
+// Define a custom log handler
+struct OTelLogHandler: LogHandler {
+    public private(set) var scope: InstrumentationScope // Property to store instrumentation scope name
+    private var loggerProvider : LoggerProvider  // Property to set LoggerProvider
+    private var logger: OpenTelemetryApi.Logger 
+
+    // Define the log level for this handler
+    private var _logLevel: Logging.Logger.Level = .info
+    public var logLevel: Logging.Logger.Level{
+        get {
+            return self._logLevel 
+        }
+        set {
+            self._logLevel = newValue
+        }
+    }
+
+    // Define metadata for this handler
+    public var metadata: Logging.Logger.Metadata = [:]
+    public subscript(metadataKey key: String) -> Logging.Logger.Metadata.Value? {
+        get {
+            return self.metadata[key]
+        }
+        set {
+            self.metadata[key] = newValue
+        }
+    }
+    
+    // should default logger provider be a noop? or the sdk implementation?
+    public init(scope: InstrumentationScope = InstrumentationScope(name: bridgename, version: version),
```

Comment:
does the instrumentation scope actually need to be settable?  Is the idea here if someone wants to use this bridge but wants to give it a different instrumentation name? Is that a necessary use case?